core/readme/src/index.ts

```diff
@@ -1,11 +1,13 @@
-import marked from 'marked';
 import createDOMPurify from 'dompurify';
 import { JSDOM } from 'jsdom';
 
 const DOMPurify = createDOMPurify(new JSDOM('').window);
+const { marked } = require('marked');
```

Comment:
Are you sure this is right?
```suggestion
import { marked } from 'marked';
```
 That should work too in typescript. 😉

Reply:
Agree :) I'll fix in, I need to upgrade lowdb https://github.com/typicode/lowdb/releases/tag/v2.0.0 anyway ...